jekyll/_cci2/docker-layer-caching.md

```diff
@@ -47,7 +47,7 @@ jobs:
   build:
     docker:
       # DLC does nothing here, its caching depends on commonality of the image layers.
-      - image: circleci/node:14.17.3-buster-browsers
+      - image: cimg/node:14.17.3-browsers
```

Comment:
I think we should just remove the browsers variant if possible, otherwise we need to include the browser testing orb in the example and it's not required to show how DLC works?

Reply:
I agree

Reply:
removed